src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -39,6 +41,8 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
             } else {
                 // make required
                 if (!is_array($value) && !is_bool($value) && 0 === strlen($value)) {
+                    @trigger_error('The default question validator is deprecated since Symfony 3.4 and will not be used anymore in version 4.0. Set a custom question validator instead.', E_USER_DEPRECATED);
```

Comment:
@ro0NL is this really in the correct code path? Throwing both a deprecation and an exception is suspicious to me.